microhapulator/data/paired.html

```diff
@@ -0,0 +1,119 @@
+{% extends "template.html" %}
```

Comment:
This `{% extends "template.html" %}'  allows the sub-templates to inherit the contents of the main `template.html`